### ISSUE TITLE
Method Ambiguous Error *(::Adjoint{Float64,Array{Float64,2}}, ::NamedDimsArray

### ISSUE THREAD

**Issue by AlexRobson** [invenia/NamedDims.jl#60]
MFE:
```
d = NamedDimsArray{(:features, :observations)}(rand(4, 10))
β = rand(4, 3)
β' * d[:, 1]
MethodError: *(::Adjoint{Float64,Array{Float64,2}}, ::NamedDimsArray{(:features,),Float64,1,Array{Float64,1}}) is ambiguous. Candidates:
  *(a::AbstractArray{T,2} where T, b::NamedDimsArray{#s19,#s20,1,A} where A<:AbstractArray{#s20,1} where #s20 where #s19) in NamedDims at /Users/alexr/.julia/packages/NamedDims/MjTg0/src/functions_math.jl:74
  *(adjA::Adjoint{#s623,#s622} where #s622<:AbstractArray{T,2} where #s623, x::AbstractArray{S,1}) where {T, S} in LinearAlgebra at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.1/LinearAlgebra/src/matmul.jl:103
Possible fix, define
  *(::Adjoint{#s623,#s622} where #s622<:AbstractArray{T,2} where #s623, ::NamedDimsArray{#s19,S,1,A} where A<:AbstractArray{S,1} where #s19)
```

Where as this is fine:
```
d = NamedDimsArray{(:features, :observations)}(rand(4, 10))
β = rand(4, 3)
collect(β') * d[:, 1]
```

**Comment by oxinabox** [comment]
Something is  missing somewhere in here
https://github.com/invenia/NamedDims.jl/blob/master/src/functions_math.jl

**Comment by mzgubic** [comment]
closed by #118
